test/cypress/cypress/support/index.js

```diff
@@ -76,9 +76,10 @@ beforeEach(() => {
     })
     cy.setSessionStorage('healthCheck', 'executed');
 
-     if (Cypress.env('type') == 'wzd') {
+   /*  if (Cypress.env('type') == 'wzd') {
```

Comment:
uncomment this, because the mock API selection will fail on execution the GitHub Actions

Reply:
Thank you !